stjude/proteinpaint
# GDC API validation fix.

## Description

Requires the `gdc-api-validation-handling` from the `ppgdc` repo.

To test:
Add GDC dataset to serverconfig.json
Execute: 
npm run dev

The server should start. 

Turn off WIFI connection and execute again:
npm run dev
The server should NOT start. 

Add to serverconfig:

```
 "features": {
    "runRemainingWithoutAwait": true
  }
```
Execute again:
npm run dev
The server should start. 



## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
